tests/unit/test_clip_text.py

```diff
@@ -0,0 +1,70 @@
+import clip
+import numpy as np
+import torch
+from jina import Flow, Document, DocumentArray, requests, Executor
+
+from jinahub.encoder.clip_text import CLIPTextEncoder
+
+
+def test_fail():
+    class MockExecutor(Executor):
+        @requests
+        def encode(self, **kwargs):
+            pass
+
+    with Flow().add(uses=MockExecutor) as f:
+        f.post(on='/test', inputs=[Document(text='whatever')])
+
+
+def test_clip_batch():
+    test_docs = DocumentArray((Document(text='random text') for _ in range(30)))
+    clip_text_encoder = CLIPTextEncoder()
+    parameters = {'default_batch_size': 10, 'model_name': 'ViT-B/32'}
+    clip_text_encoder.encode(test_docs, parameters)
```

Comment:
This test shows nothing, check ur code. There are only 2 parameters read

Reply:
The other params get default values. And the test shows that with a batch_size of 10 and total data size of 30, all text data are encoded properly ?

Reply:
batch size is not being overriden here

Reply:
fixing